packages/automator/README.md

```diff
@@ -39,3 +41,8 @@ In addition to batch-processing Penrose programs, you can also use `automator` t
 - Open `browser/index.html` to view the result.
 
 ![](docs/penrose-artifacts.png)
+
+## 06/25/2021 Update: Staged Diagram Generation
```

Comment:
Don't have to date this :P. A quick screenshot would be great!

Reply:
What do you mean by a screenshot? 😅